### PR TITLE
Add "Highest measurement" on bGeigie import detail page

### PR DESCRIPTION
**Pull request by eitoball**
Connected to #245.

bgeigie_import__1____bgeigie_imports____safecast_api

### CODE REVIEW

File: app/views/bgeigie_imports/_static_metadata.html.erb [hunk 1]
```diff
@@ -1,28 +1,14 @@
...
-    </dd>
+    <dt><%= t('.uploaded_by') %></dt>
+    <dd><%= @bgeigie_import.user.name_or_email %></dd>
+    <dt><%= t('.filename') %></dt>
+    <dd><%= @bgeigie_import.source.filename %></dd>
+    <dt><%= t('.number_of_lines') %></dt>
+    <dd><%= @bgeigie_import.lines_count %></dd>
+    <dt><%= t('.number_of_measurements') %></dt>
+    <dd><%= @bgeigie_import.processed? ? @bgeigie_import.bgeigie_logs.size : '-' %></dd>
+    <dt>Highest Measurement</dt>
```
Comment (matschaffer) [inline app/views/bgeigie_imports/_static_metadata.html.erb]: Seems like we might not want to break the translation pattern here. Mind at least dropping it into the `en` strings file?

File: app/views/bgeigie_imports/_static_metadata.html.erb [hunk 2]
```diff
@@ -1,28 +1,14 @@
...
+    <dd><%= @bgeigie_import.maximum_cpm || '-' %></dd>
```
Comment (matschaffer) [inline app/views/bgeigie_imports/_static_metadata.html.erb]: Looks like the show page isn't bad anymore, but could be good to check on [newrelic](https://rpm.newrelic.com/accounts/1057812/applications/10580222/transactions?type=app#id=5b22436f6e74726f6c6c65722f626765696769655f696d706f7274732f73686f77222c22225d) once this is deployed to ensure the new `max()` call doesn't cause undue load.

File: app/views/bgeigie_imports/_static_metadata.html.erb (hunk 1 above)
Comment (eitoball) [reply app/views/bgeigie_imports/_static_metadata.html.erb]: Yep, done.

File: app/views/bgeigie_imports/_static_metadata.html.erb (hunk 2 above)
Comment (matschaffer) [reply app/views/bgeigie_imports/_static_metadata.html.erb]: Newrelic is looking clean. And also looks like that page doesn't see massive load anyway, so I'd say we're 👍 

screen shot 2018-07-12 at 8 22 47

Comment (eitoball) [reply app/views/bgeigie_imports/_static_metadata.html.erb]: Thank you for checking out.

**Pull request merged**